### ISSUE TITLE
[FEATURE REQUEST] Separate Notification Icon

### ISSUE THREAD

**Issue by moseskarunia** [fluttercommunity/flutter_launcher_icons#219]
### :speech_balloon: Description
In android usually apps uses the white icon version for the notification, instead of the default app icon. This usually makes the notification icon displayed better on an always-on display, like samsung's phone

This library doesn't necessarily need to automatically generate the image itself (though it's stellar if can be done automagically), but instead expose another option under the config like:

```
flutter_icons:
  android: true
  ios: true
  image_path: "assets/app_icons/ios_app_icon.jpg"
  adaptive_icon_foreground: "assets/app_icons/adaptive_foreground.png"
  adaptive_icon_background: "#000000"
  notification_android: "assets/app_icons/white_app_icon.svg" // requested feature
```

### :question: Platform
I don't know about other platform, but afaik, only in android we are able to do things like this. In ios, notification icon is always equal to the app icon.

### Expected Behavior
When run, will generate xml file under android drawable from a given svg.

**Comment by MaddinMade** [comment]
#291

**Comment by MaddinMade** [comment]
This feature was implemented in this issue: #291
